src/passes/Print.cpp

```diff
@@ -3442,6 +3442,15 @@ printStackIR(StackIR* ir, std::ostream& o, Function* func) {
   return o;
 }
 
+std::ostream& printStackIR(std::ostream& o, Module* module) {
+  wasm::PassRunner runner(module);
+  Pass* generatePass = createGenerateStackIRPass();
```

Comment:
Instead of this, you can do `runner.add(..)` to create and add the pass, then do `runner.run()`. That would avoid manual memory management here.

Reply:
Have modified as you suggested